src/Features/LanguageServer/Protocol/Features/CodeFixes/CodeFixService.cs

```diff
@@ -664,6 +665,7 @@ private bool TryGetWorkspaceFixersPriorityMap(TextDocument document, [NotNullWhe
             // append CodeFixCollection for each CodeFixProvider
             foreach (var provider in lazyConfigurationProviders.Value)
             {
+                using (Logger.LogBlock(FunctionId.CodeFixes_GetCodeFixesAsync, KeyValueLogMessage.Create(LogType.UserAction, m => CreateLogProperties(m, provider)), cancellationToken))
```

Comment:
if we move logging into `TryGetFixesOrConfigurationsAsync` then I do not think this log statment is necessary.